bug: Enforce Repository Settings workflow fails with 'gh is not authenticated'

## Problem

The `Enforce Repository Settings` workflow fails on push to `main` with:

```
[ERROR] gh is not authenticated
```

This occurs in Phase 1 (Validate), suggesting the `REPO_SETTINGS_TOKEN` secret is missing, expired, or lacks the required permissions.

## Workflow run

https://github.com/f5xc-salesdemos/docs-control/actions/runs/22075189523

## Impact

Repository settings are not being self-enforced on docs-control when changes are pushed to `main`. Downstream repos are unaffected (the Dispatch Downstream workflow uses its own token and succeeds).

## Discovered during

CI monitoring after merging PR #103.

--- a/.github/actionlint.yaml
+++ b/.github/actionlint.yaml
@@ -8,3 +8,6 @@ paths:
   .github/workflows/**/*.yml:
     ignore:
       - "SC2016:.+"
+  .github/workflows/enforce-repo-settings.yml:
+    ignore:
+      - 'property "repo_settings_token" is not defined'

--- a/.github/workflows/enforce-repo-settings.yml
+++ b/.github/workflows/enforce-repo-settings.yml
@@ -24,7 +24,7 @@ jobs:
 
       - name: Enforce repository settings
         env:
-          GH_TOKEN: ${{ secrets.repo-settings-token }}
+          GH_TOKEN: ${{ secrets.repo-settings-token || secrets.REPO_SETTINGS_TOKEN }}
         run: |
           set -euo pipefail
 

